enlighten/assets/example_data/mock_raman/mocks.json

```diff
@@ -0,0 +1,22 @@
+{
+  "Cyclohexane": {
+    "peak_location_cm": [ 392, 437, 804, 1031, 1165, 1273, 1358, 1453 ],
```

Comment:
These are close, but not the correct values?
![Cyclohexane](https://user-images.githubusercontent.com/2125705/204351507-7c13a046-7d44-4bf3-858a-4b2b65322815.png)